Classes/Domain/AbstractEventSourcedAggregateRoot.php

```diff
@@ -42,8 +42,8 @@
     public static function reconstituteFromEventStream(string $identifier, EventStream $stream)
     {
         $instance = new static($identifier);
-        foreach ($stream as $eventWithMetadata) {
-            $instance->apply($eventWithMetadata->getEvent());
+        foreach ($stream as $eventTransport) {
+            $instance->apply($eventTransport->getEvent());
```

Comment:
But does it have to communicate *what it is for*? Shouldn't rather tell *what it is* or *what it contains*?